skipper.go

```diff
@@ -802,6 +811,20 @@ func createDataClients(o Options, auth innkeeper.Authentication) ([]routing.Data
 		}
 	}
 
+	if len(o.RemoteRoutesURLs) > 0 {
+		for _, url := range o.RemoteRoutesURLs {
+			client, err := eskipfile.RemoteWatch(&eskipfile.RemoteWatchOptions{
+				RemoteFile:    url,
+				FailOnStartup: o.RemoteRoutesFailOnStartup,
+			})
+			if err != nil {
+				log.Errorf("error while loading remote eskip file from url %s: %s", url, err)
```

Comment:
lets remove `remote` and `file` from the message, e.g. `error while loading routes from url %s: %s`